src/Package/Impl/DataInspect/Viewers/ViewerBase.cs

```diff
@@ -3,6 +3,7 @@
 
 using System.Threading.Tasks;
 using Microsoft.R.Components.Extensions;
+using Microsoft.R.DataInspection;
 using Microsoft.R.Debugger;
```

Comment:
Yep. I'll do a sweep - there shouldn't be anything using it anymore. In fact, it shouldn't even be declared.

---
In reply to: [62136426](https://github.com/Microsoft/RTVS/pull/1731#discussion_r62136426) [](ancestors = 62136426)